content/docs/1.7.0/advanced-resources/os-distro-specific/container-optimized-os-support.md

```diff
@@ -0,0 +1,67 @@
+---
+title:  Container-Optimized OS (COS) Support
+weight: 5
+---
+
+## Requirements
+
+> **Note:**
+> Longhorn currently supports Container-Optimized OS only when used as the base image for Google Kubernetes Engine (GKE), which includes a pre-configured Kubernetes environment. The following information may not apply to manually created Kubernetes environments, including Kubernetes provisioned with other orchestrators.
+
+The Container-Optimized OS does not include a package manager and does not allow non-containerized applications to run. Additionally, its root filesystem is mounted as read-only, which poses a challenge for IO operations.
+
+In GKE, Kubernetes tackles these constraints by housing necessary dependencies in a chroot environment (`/home/kubernetes/containerized_mounter/rootfs`) and mounting directories within it, enabling the execution of required tasks.
+
+Longhorn provides a GKE COS node agent daemonset, which leverages GKE Kubernetes solutions to configure and run necessary dependencies. This agent is responsible for the following operations:
```

Comment:
Should we add `GKE COS node agent` into the scope of the tooling cli?

Reply:
https://github.com/longhorn/longhorn/issues/7927